src/werkzeug/datastructures.py

```diff
@@ -1844,6 +1844,19 @@ def _normalize(language):
 
         return item == "*" or _normalize(value) == _normalize(item)
 
+    def best_match(self, matches, default=None):
+        """Calls the super version of `best_match` and if it returns none,
+        attempts to fall back to language only matches.
+
+        :param matches: a list of matches to check for
+        :param default: the value that is returned if none match
+        """
+        result = super(LanguageAccept, self).best_match(matches, default=default)
+        if result is not None:
+            return result
+        fallback = Accept([(item[0][0:2], item[1]) for item in list(self)])
```

Comment:
(Minor) You can use `Accept((item[0][0:2], item[1]) for item in self)` to avoid creating unnecessary temporary lists.

Reply:
Done. Thanks for pointing that out.